### DISCUSSION TITLE
Android/Apple Watch companion app

### DISCUSSION THREAD

**Discussion by RP-96** [standardnotes/forum#3135] [Suggestions]
With the ever increasing sales of wearable devices I'd like to suggest having a companion app to be able to view notes.

Maybe something simple where you can only view your notes. Google Notes is a very good companion app on the Galaxy Watch. I would prefer not to use it but as there isn't a companion app for my preferred app choice (Standard Notes) so I am not left with much choice.

**Comment by nicholasrebstock** [comment]
I would most want to use this for my shopping list so that I can see it while I'm out without my phone. The ability to edit would be useful for a shopping list but even view only would be helpful.
